File: blog-dao/src/main/java/com/dao/ClickMapper.java
package com.dao;

import com.entity.Click;

public interface ClickMapper {
    int deleteByPrimaryKey(Integer clid);

    int insert(Click record);

    int insertSelective(Click record);

    Click selectByPrimaryKey(Integer clid);

    int updateByPrimaryKeySelective(Click record);

    int updateByPrimaryKey(Click record);

    //通过文章id查询所对应的点击量
    Click selectArticleId(Integer aid);
}

File: blog-dao/src/main/java/com/dao/ClickMapper.kt
package com.dao

import com.entity.Click

interface ClickMapper {
    fun deleteByPrimaryKey(clid: Int?): Int

    fun insert(record: Click): Int

    fun insertSelective(record: Click): Int

    fun selectByPrimaryKey(clid: Int?): Click

    fun updateByPrimaryKeySelective(record: Click): Int

    fun updateByPrimaryKey(record: Click): Int

    //通过文章id查询所对应的点击量
    fun selectArticleId(aid: Int?): Click
}